package.json

```diff
@@ -10,11 +10,19 @@
     "url": "git@github.com:BBVAEngineering/ember-cli-htmlbars-minifier.git"
   },
   "scripts": {
-    "pretest": "npm run check && npm run lint",
-    "test": "npm run mocha",
-    "lint": "node_modules/.bin/eslint '**/*.js'",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint --max-warnings 0 ./*.js lib tests",
     "check": "node_modules/.bin/check-dependencies",
-    "mocha": "node_modules/.bin/mocha --recursive tests"
+    "pretest": "npm run check && npm run lint:js",
+    "test": "npm run clean-coverage && npm run test:node",
```

Comment:
`clean-coverage` should be added to `pretest`.

Reply:
moved!